core/src/main/scala/org/apache/spark/internal/config/package.scala

```diff
@@ -2255,10 +2255,17 @@ package object config {
       .checkValue(_ >= 0, "needs to be a non-negative value")
       .createWithDefault(5)
 
+  private[spark] val STAGE_MAX_CONSECUTIVE_ATTEMPTS =
+    ConfigBuilder("spark.stage.maxConsecutiveAttempts")
+      .doc("Number of consecutive stage attempts allowed before a stage is aborted.")
+      .version("2.2.0")
+      .intConf
+      .createWithDefault(4)
```

Comment:
Mark this `internal` ?

Reply:
Looks like it's a public configuration item since 2.2.0.

https://spark.apache.org/docs/latest/configuration.html

https://github.com/apache/spark/blob/9b43a9f3ea551a594835a4742f7b2d1fdb1cf518/docs/configuration.md?plain=1#L2912-L2919

Reply:
Thanks for checking ! Looks like my search string had unnecessary whitespace at the end :-(